File: web-nodejs/extension-common/src/model/FlexMediaDetailSeries.ts
import {FlexMediaSectionItem} from "./FlexMediaSectionItem";

/**
 * 条目的详情页的相关系列的板块
 *
 * @author why
 * @since 5/8/24
 */
export interface FlexMediaDetailSeries {
    /**
     * 标题
     */
    title: string

    /**
     * 当前条目的ID
     */
    mediaId: string

    /**
     * 数目
     */
    count: number

    /**
     * 系列板块包含的小条目
     */
    items: FlexMediaSectionItem[]
}

File: lib-extension/src/main/java/com/xiaoyv/flexiflix/extension/java/model/FlexMediaDetailSeries.kt
package com.xiaoyv.flexiflix.extension.java.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * [FlexMediaDetailSeries]
 *
 * @author why
 * @since 5/8/24
 */
@Parcelize
data class FlexMediaDetailSeries(
    var title: String = "默认",
    var mediaId: String,
    var count: Int = 0,
    var items: List<FlexMediaSectionItem> = emptyList()
) : Parcelable
